gio/src/socket_msg_flags.rs

```diff
@@ -0,0 +1,77 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+use glib::value::{FromValue, ToValue};
+use glib::{StaticType, Type};
+use std::fmt;
+
+bitflags::bitflags! {
+    #[doc(alias = "GSocketMsgFlags")]
```

Comment:
Why can't this be autogenerated?

Reply:
Gir thinks all flags are `u32`, but this is a `c_int`

Reply:
Ah I see. Sucks but makes sense